editor/src/components/inspector/inspector-common.ts

```diff
@@ -516,6 +516,12 @@ export const nukeAllAbsolutePositioningPropsCommands = (
   ]
 }
 
+export const nukeSizingProps = (path: ElementPath): Array<CanvasCommand> => {
```

Comment:
We have `nukeSizingPropsForAxisCommand` which also includes the `min*` and `max*` props, so maybe it's worth either calling that for both axes, or including those props here

Reply:
fixed in https://github.com/concrete-utopia/utopia/pull/3510/commits/08051de7ece2a962fb4cb23735391b9009e75363